modules/ROOT/pages/to-share-an-asset.adoc

```diff
@@ -0,0 +1,79 @@
+
+. Click an asset to show its details.
+. Click *Share*.
+. Begin typing the name of your organization.
+. Select the list item for everyone at your organization.
+. Select whether to grant Viewer, Contributor, or Admin rights.
+. Click *Save*.
+
+When an asset is shared with an organization, email notification is sent to the organization administrators.
+
+You can also click *Share* to stop sharing the asset with everyone at the organization.
+
+== Share With an External Organization
+
+Exchange administrators and asset administrators can grant viewer access for an asset to everyone in a chosen external organization.
+
+The organization granting viewer access is the granting organization, and the organization that receives access is the receiving organization.
+
+. The receiving organization adds the granting organization to its "External Access" list in Anypoint Access Management, as described in xref:access-management::external-organization-access.adoc[External Organization Access].
++
+Assets can only be shared with another organization if that organization has enabled receiving viewer access to assets from external organizations.
++
+. The granting organization adds the receiving organization to its "External Access" list in Anypoint Access Management, as described in xref:access-management::external-organization-access.adoc[External Organization Access].
+. Login to your Exchange administrator or asset administrator account in the granting organization.
+. Click an asset to show its details.
+. Click *Share*.
+. Begin typing the name of the external organization you want to share this asset with.
+. Select the organization in the list.
+. Repeat steps 6 and 7 to share with additional organizations.
+. Click *Save*.
+
+When an asset is shared with an organization, email notification is sent to the organization administrators.
+
+You can also click *Share* to change who you are sharing the asset with or to stop sharing with one or more organizations.
+
+Only internal users in the same organization as the asset can request access to an API or gain the contributor or administrator roles.
+
+=== Using Assets Shared with External Organizations
+
+If a connector is shared with an external organization, then in Studio or Flow Designer a user in the external organization can search for the connector, configure it, and use it in a Mule application.
+
+If an example or template is shared with an external organization, then in Studio or Flow Designer a user in the external organization can implement a Mule application based on the shared template.
+
+If a REST, SOAP, or HTTP API is created in Exchange and shared with an external organization, then API Manager cannot manage the API in the external organization. API Manager search results show only APIs from the external organization user's business group. In Studio a user in the external organization can use the API in the flow as a source for scaffolding an application.
```

Comment:
Is this correct?
```suggestion
If a REST, SOAP, or HTTP API is created in Exchange in one organization and then shared with an external organization, API Manager cannot manage the API in that external organization. API Manager search results show only APIs from the external organization user's business group. However, a user in the external organization can use Studio to use the API in the flow, as a source for scaffolding an application.
```
The second sentence troubles me because it seems to unnecessarily complicate matters. We are describing sharing and use, right, and not so much searching and confirming? Also, does this sentence mean that API Manager shows only APIs already existing in the receiving organization, and not anything shared.

Reply:
Looking forward, it appears that the focus of the topic has changed a bit, from just using the shared asset to using it but with restrictions in managing it with API Manager. Is that right?
This is pretty complicated!
I wonder if some sort of illustration might help.

Reply:
Quite complicated!

Rewrote to emphasize the use, and indicate that searching is secondary.

API Manager does show only APIs already existing in the receiving organization, and not anything shared.

Assets are shared with Exchange so that they can be used in other products. This final section talks about the various products that can and cannot use these shared assets, and in what ways.